static/app/components/forms/fields/choiceMapperField.tsx

```diff
@@ -125,7 +125,10 @@ export default class ChoiceMapperField extends Component<ChoiceMapperFieldProps>
     } = props;
 
     const mappedKeys = Object.keys(columnLabels);
-    const emptyValue = mappedKeys.reduce((a, v) => ({...a, [v]: null}), {});
+    const emptyValue = mappedKeys.reduce((a, v) => {
+      a[v] = null;
+      return a;
+    }, {});
```

Comment:
Alternatively: 
```suggestion
    const emptyValue = Object.fromEntries(mappedKeys.map(key => ([key, null])));
```

Reply:
Oh nice, I like that. I think we had some errors around Object.fromEntries in unsupported browsers though, I'd need to check